roles/candlepin/tasks/artemis.yml

```diff
@@ -7,42 +7,32 @@
   ansible.builtin.set_fact:
     candlepin_artemis_client_dn: "{{ openssl_response.stdout | replace('subject=', '') }}"
 
-- name: Copy Candlepin broker.xml
-  ansible.builtin.template:
-    src: broker.xml.j2
-    dest: /etc/candlepin/broker.xml
-    owner: root
-    group: tomcat
-    mode: '0640'
+- name: Create Candlepin broker.xml
+  containers.podman.podman_secret:
+    state: present
+    name: broker.xml
```

Comment:
should this be `candlepin-broker.xml`? or, more generally, `${role}-${file}`?

gives us a bit of a namespace (imagine two services having a settings.yaml) and ease the "huh, where the f does this secret come from" when you look at `podman secrets ls`

Reply:
Yea - that's fair. Take a look at the example here: https://github.com/ehelms/foreman-quadlet/pull/16/files#diff-6e185dcb895442d3f858c32b6dbd8fbe1854d41eeabe02378b89cc7fa0d0cca3R22-R26

I was thinking context will be helpful, and that we can use Labels to provide context. These are not visible through `podman secret ls` but they are visible when using `podman secret inspect`. I captured my thoughts of conventions to start:

https://github.com/ehelms/foreman-quadlet/pull/16/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R47-R62

Reply:
I like that! Commented some bikeshedding, but that doesn't need an absolute resolution just now.